datalake/src/main/java/com/sequenceiq/datalake/flow/create/handler/RdsWaitHandler.java

```diff
@@ -101,18 +101,20 @@ private void setRdsCreatedStatus(SdxCluster cluster) {
 
     private void validForDatabaseCreation(Long sdxId, DetailedEnvironmentResponse env) {
         String message;
-        if (env.getNetwork().getSubnetMetas().size() < 2) {
-            message = String.format("Cannot create external database for sdx: %s, not enough subnets in the vpc", sdxId);
-            LOGGER.debug(message);
-            throw new BadRequestException(message);
-        }
-        Map<String, Long> zones = env.getNetwork().getSubnetMetas().values().stream()
-                .collect(Collectors.groupingBy(CloudSubnet::getAvailabilityZone, Collectors.counting()));
-        if (zones.size() < 2) {
-            message = String.format("Cannot create external database for sdx: %s, the subnets in the vpc should be at least in two different availabilityzones",
-                    sdxId);
-            LOGGER.debug(message);
-            throw new BadRequestException(message);
+        if ("aws".equalsIgnoreCase(env.getCloudPlatform())) {
```

Comment:
Right, I think it is redundant. I decided to leave it in place for now, because it doesn't hurt, and the goal of the jira is to focus on Azure, not AWS.